src/utils/TriangleUtils.ts

```diff
@@ -61,56 +133,196 @@ let cleanIndex = function (ind, lengthArray) {
         res = ind % lengthArray;
     }
     return res;
-};
+},
 
 /**
- * A number, or a string containing a number.
- * @typedef {Object} VertexLike
- * @property {() => Vector3} getPos
- * @property {() => number} getThickness
+ * Updates the cached values of the triangle
+ * @param triangle The triangles who's internal values need to be updated
  */
+updateComputedAttributes(triangle: TriangleLike) {
```

Comment:
Where does this function come from ?
Diff seems to be broken here I cant review that

Reply:
This function is new, it is here to replace computeVectorDirs as it is more general in its typing